frontend/src/scenes/organization/Settings/index.tsx

```diff
@@ -103,6 +103,39 @@ function DomainWhitelist({ isRestricted }: RestrictedComponentProps): JSX.Elemen
     )
 }
 
+function EmailPreferences({ isRestricted }: RestrictedComponentProps): JSX.Element {
+    const { currentOrganization, currentOrganizationLoading } = useValues(organizationLogic)
+    const { updateOrganization } = useActions(organizationLogic)
+
+    return (
+        <div>
+            <h2 id="name" className="subtitle">
+                Notification Preferences
+            </h2>
+            <div>
+                <Switch
+                    // @ts-expect-error - id works just fine despite not being in CompoundedComponent
+                    id="is-member-joined-email-enabled"
+                    onChange={(checked) => {
+                        updateOrganization({ is_member_join_email_enabled: checked })
+                    }}
+                    checked={currentOrganization?.is_member_join_email_enabled}
+                    loading={currentOrganizationLoading}
+                    disabled={isRestricted || !currentOrganization}
+                />
+                <label
+                    style={{
+                        marginLeft: '10px',
+                    }}
+                    htmlFor="is-member-joined-email-enabled"
```

Comment:
Q: Would nesting this in the `<Switch>` work? This would get rid of the `id` requirement / `@ts-expect-error`

Reply:
Unfortunately `<Switch>` does not support `children`.